packages/components/package.json

```diff
@@ -28,7 +26,7 @@
     "@std-proposal/temporal": "0.0.1",
     "@types/color": "^3.0.1",
     "@types/lodash": "^4.14.136",
-    "@types/react-datepicker": "^4.1.7",
+    "@types/react-datepicker": "^4.3.2",
```

Comment:
Kind of just thinking out loud here, but confirmation might be cool.  IIRC When I worked on this with Eddy the patch was to get typing for the datepicker.  So I assume upgrading it takes care of that?

Reply:
Yup. Can test it like this, if the patch ain't there, typescript will get angry at you.